code/modules/cargo/packs/sec_supply.dm

```diff
@@ -90,19 +90,25 @@
 	crate_name = "combat knife crate"
 */
 
-/datum/supply_pack/sec_supply/fire
-	name = "Incendiary Weapons Crate"
-	desc = "Burn, baby burn. Contains three incendiary grenades and three prebuilt flamethrowers."
-	cost = 5000
+/datum/supply_pack/sec_supply/flamethrower
+	name = "Flamethrower Crate"
+	desc = "Contains one flamethrower. Point the nozzle away from anything important."
+	cost = 1250
+	contains = list(
+		/obj/item/flamethrower/full
+	)
+	crate_name = "flamethrower crate"
+	crate_type = /obj/structure/closet/crate/secure/weapon
+	faction = FACTION_NGR
+
+/datum/supply_pack/sec_supply/incendiary_grenade
+	name = "incendiary Grenade Crate"
```

Comment:
```suggestion
	name = "Incendiary Grenade Crate"
```

Reply:
Thanks for catching that! Let me add the changes in